Create resources.md

rachel-kauff opened the pull request:
- Add section to Diversity-Equity-Inclusion-Practice-Area for educational resources. This section will be linked in the DEI onboarding card.

rachel-kauff committed:
Create resources.md

- Add section to Diversity-Equity-Inclusion-Practice-Area for educational resources. This section will be linked in the DEI onboarding card.

erogray reviewed (changes requested):
Recommend changing Zoom link to permission'd slides link

rachel-kauff committed:
Update Resources.md

- Changed link on line 14 to the slide deck (which has a link to the ahc recording within)
